routes/router.js

```diff
@@ -87,12 +87,10 @@ module.exports = function(app, passport, path) {
 	  	function(req, res) {
 	  		console.log(JSON.stringify(req.user));
 	  		if (!req.user) {
-	  			// res.redirect('/login');
```

Comment:
You'll need to send some kind of response - may be just an error status like you have for other error cases